.travis.yml

```diff
@@ -20,7 +20,6 @@ env:
     - TARGET_PLATFORM=debian/stretch
     - TARGET_PLATFORM=ubuntu/bionic
```

Comment:
@JelteF can you go over the list of distros here, and add some missing ones if there are any?

**el/6** may have some outdated tooling and create issues
**ol/6** is a bit old, and I do not know why we do not have it here
**ol/8** it is unclear if PackageCloud supports this distro yet.
**Ubuntu/focal** is relatively new and I guess should be added here.

Reply:
Tried with the these distros, they all fail: https://travis-ci.org/github/citusdata/packaging/builds/683750588

1. el/6 and ol/6 fail because of a static assert that is unsupported by their ancient gcc. This should be fixed for el/6 once we build new pg10 images, since it will use a new GCC then.
2. ol/8 and ubuntu/focal fail because of missing images on docker

Reply:
> ol/8 and ubuntu/focal fail because of missing images on docker.

I forgot to push the images, sorry. Do you think we should create the packages for those in a separate PR?

Thanks for spending the time to make sure we do not miss anything.